source/common/router/router.h

```diff
@@ -285,6 +301,9 @@ class Filter : Logger::Loggable<Logger::Id::router>,
       stream_info_.onUpstreamHostSelected(host);
       upstream_host_ = host;
       parent_.callbacks_->streamInfo().onUpstreamHostSelected(host);
+      if (parent_.retry_state_ && host) {
+        parent_.retry_state_->onHostAttempted(host);
```

Comment:
Is this the only place we call this now?

Reply:
yeah